wiki/People/en.md

```diff
@@ -0,0 +1,13 @@
+# People
+
+Members of the osu!community that mantian *osu!*
```

Comment:
change to `The following are user groups, made up of the osu!community members that help maintain _osu!_`